### DISCUSSION TITLE
Is there a way to remove focus on an CTkEntry field?

### DISCUSSION THREAD

**Discussion by Finn-Kramer** [TomSchimansky/CustomTkinter#1726] [General]
Hello i have a question about the CTkEntry!
i want the text inputed into the field being shown when pressing Enter on the Keyboard. i used to do this by setting the focus to a different UI object (a CTkButton). now this solution does not work anymore so i was wondering if there is a different solution to remove focus on a widget!

![grafik](https://user-images.githubusercontent.com/119854167/224483640-929bde88-d7b6-4a30-a144-fd5460b66e7e.png)

after pressing enter:
![grafik](https://user-images.githubusercontent.com/119854167/224483657-78b0734e-2d19-4817-9e2d-eaf3c3e81876.png)

i have to click out of the window or a different button to make show the text entered before a Replacement text again:
![grafik](https://user-images.githubusercontent.com/119854167/224483690-786a6662-4822-40c3-ae49-e26c11880897.png)

**Comment by Akascape** [comment]
@Finn-Kramer 
Just add this line:
```python
root.bind("<1>", lambda event: event.widget.focus_set())
```
*here root is the customtkinter.CTk() window*
